base_layer/core/src/base_node/consts.rs

```diff
@@ -23,6 +23,12 @@
 use std::time::Duration;
 
 /// The allocated waiting time for a request waiting for service responses from remote base nodes.
-pub const BASE_NODE_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(180);
+pub const GENERAL_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(180);
```

Comment:
These can be removed since they are defaulted in the config

Reply:
Constants removed in favour of moving the values directly into the default struct method.